src/scitiff/_resources/metadata-schema.json

```diff
@@ -0,0 +1,383 @@
+{
+  "$defs": {
+    "ImageDataArrayMetadata": {
+      "description": "Image DataArray Metadata without values(image).",
+      "properties": {
+        "data": {
+          "$ref": "#/$defs/ImageVariableMetadata"
+        },
+        "masks": {
+          "additionalProperties": {
+            "$ref": "#/$defs/ScippVariable"
+          },
+          "title": "Masks",
+          "type": "object"
+        },
+        "coords": {
+          "additionalProperties": {
+            "$ref": "#/$defs/ScippVariable"
+          },
+          "title": "Coords",
+          "type": "object"
+        },
+        "name": {
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "null"
+            }
+          ],
+          "default": null,
+          "title": "Name"
+        }
+      },
+      "required": [
+        "data"
+      ],
+      "title": "ImageDataArrayMetadata",
+      "type": "object"
+    },
+    "ImageVariableMetadata": {
+      "description": "Image Metadata.",
+      "properties": {
+        "dims": {
+          "maxItems": 5,
+          "minItems": 5,
+          "prefixItems": [
+            {
+              "const": "c",
+              "type": "string"
+            },
+            {
+              "const": "t",
+              "type": "string"
+            },
+            {
+              "const": "z",
+              "type": "string"
+            },
+            {
+              "const": "y",
+              "type": "string"
+            },
+            {
+              "const": "x",
+              "type": "string"
+            }
+          ],
+          "title": "Dims",
+          "type": "array"
+        },
+        "shape": {
+          "maxItems": 5,
+          "minItems": 5,
+          "prefixItems": [
+            {
+              "type": "integer"
+            },
+            {
+              "type": "integer"
+            },
+            {
+              "type": "integer"
+            },
+            {
+              "type": "integer"
+            },
+            {
+              "type": "integer"
+            }
+          ],
+          "title": "Shape",
+          "type": "array"
+        },
+        "unit": {
+          "title": "Unit",
+          "type": "string"
+        },
+        "dtype": {
+          "title": "Dtype",
+          "type": "string"
+        },
+        "variances": {
```

Comment:
Why do you allow the variances to be a lower dimensional array than the Variable itself? (You require 5 dims above.)

Reply:
Ah ha it's because I didn't update it in the subclass. Thanks...!